src/screens/SliderInput.js

```diff
@@ -15,14 +15,30 @@ export default class ClassSlider extends Component {
       metric: 0,
       notification: {},
       survey: [],
-      loading: true
+      loading: true,
+      ind: 1
 
     }
   }
 
+  forceRemount = () => {
+    let ind = this.state.ind++;
+    console.log(ind)
+    this.setState({index: ind});
+    console.log(this.state)
+  }
+
   async componentDidMount() {
-    let survey = await API.get('surveysCRUD', `/surveys/random`);
-    this.setState({ survey, loading: false });
+    this._sub = this.props.navigation.addListener(
+      'didFocus',
```

Comment:
What does 'didFocus' mean?

Reply:
awesome comments @vvvli !

Reply:
DidFocus is a listener that detects if the 'tab' is 'focused'. Basically, when a user taps on an icon, that is a focus.